snapcraft.yaml

```diff
@@ -269,11 +268,11 @@ parts:
   at-spi2-core:
     after: [ atk, meson-deps ]
     source: https://gitlab.gnome.org/GNOME/at-spi2-core.git
-    source-tag: 'AT_SPI2_CORE_2_44_1'
+    source-tag: '2.50.2'
```

Comment:
Mmm... are you sure about this?
![imagen](https://github.com/ubuntu/gnome-sdk/assets/105285003/3b869c09-59fa-4917-97cc-32033b7da350)

Reply:
![imagen](https://github.com/ubuntu/gnome-sdk/assets/105285003/90f47977-89ef-4a82-b4d4-7c0dbbe4dcc7)

Reply:
Thanks, I got confused by the webUI of /tags. I've changed at-spi back to the right tag format and forced pushed